op-mode-definitions/show-isis.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="isis">
+        <children>
+          <node name="database">
+            <properties>
+              <help>Show IS-IS link state database</help>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis database detail"</command>
+              </leafNode>
+            </children>
+            <command>/usr/bin/vtysh -c "show isis database"</command>
+          </node>
+          <tagNode name="database">
+            <properties>
+              <help>Show IS-IS link state database PDU</help>
+            </properties>
+            <command>/usr/bin/vtysh -c "show isis database $4"</command>
+          </tagNode>
+          <leafNode name="hostname">
+            <properties>
+              <help>Show IS-IS dynamic hostname mapping</help>
+            </properties>
+            <command>/usr/bin/vtysh -c "show isis hostname"</command>
+          </leafNode>
+          <node name="interface">
+            <properties>
+              <help>Show IS-IS interfaces</help>
+                <completionHelp>
+                  <script>${vyos_completion_dir}/list_interfaces.py</script>
+                </completionHelp>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis interface detail"</command>
+              </leafNode>
+            </children>
+            <command>/usr/bin/vtysh -c "show isis interface"</command>
+          </node>
+          <tagNode name="interface">
+            <properties>
+              <help>Show specific IS-IS interface</help>
+                <completionHelp>
```

Comment:
`completionHelp` should be moved two to the left as it's on the same XML level as `<help>`.

Reply:
Will make change.